lib/chef-dk/chef_runner.rb

```diff
@@ -51,21 +55,25 @@ def run_context
     def policy
       return @policy_builder if @policy_builder
 
-      @policy_builder = Chef::PolicyBuilder::Dynamic.new("chef-dk", ohai.data, {}, nil, formatter)
+      @policy_builder = Chef::PolicyBuilder::Dynamic.new("chef-dk", ohai.data, {}, nil, event_dispatcher)
       @policy_builder.load_node
       @policy_builder.build_node
       @policy_builder.node.run_list(*run_list)
       @policy_builder.expand_run_list
       @policy_builder
     end
 
-    def formatter
+    def event_dispatcher
       @formatter ||=
```

Comment:
Might as well rename `@formatter` to `@event_dispatcher` - I think it is more clear